harambe/core.py

```diff
@@ -96,6 +96,7 @@ def __init__(
             else None
         )
         self._saved_data: set[ScrapeResult] = set()
+        self._saved_cookies = None
```

Comment:
this is fine as a list/set. We should append to it as we add more

Reply:
will we need to call the function more than once in the single cookies extractor?
just asking cus in this case, I will need to add logic to handle duplicated cookies